example/lib/main.dart

```diff
@@ -86,30 +98,64 @@ class _MyHomePageState extends State<MyHomePage> {
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
             Text(
-              'You have pushed the button this many times:',
+              'Status:',
+            ),
+            Text(
+              '$_status',
+              style: Theme.of(context).textTheme.bodyText1,
+              key: Key('status'),
+            ),
+            Text(
+              'Output Path:',
+            ),
+            Text(
+              '$_outputPath',
+              style: Theme.of(context).textTheme.bodyText1,
+              key: Key('output_path'),
             ),
             Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+              'Output File Size:',
+            ),
+            Text(
+              '$_outputFileSize',
+              style: Theme.of(context).textTheme.bodyText1,
+              key: Key('output_file_size'),
+            ),
+            RaisedButton(
+              key: Key('select_video'),
+              onPressed: () async {
+                File file =
+                    await ImagePicker.pickVideo(source: ImageSource.gallery);
```

Comment:
do we really need ImagePicker and mocking logic ? how about just get from `File(path)`

Reply:
ImagePicker is useful if you're using the example app interactively. I can have a different button to read from bundle instead.

Reply:
Purpose of this library is to compress video, there is no picker in this lib

Input -> Video file, compress() then Expect -> compressedVideo and test remove cache

Reply:
yes there is no picker in this lib, but this example app is actually for both testing and trying it out interactively.

I've already removed the image picker from the test flow.